.github/workflows/ci.yml

```diff
@@ -42,14 +42,9 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '11.0.0'
-      - name: NVM
-        run: |
-          curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
-          export NVM_DIR="$HOME/.nvm"
-          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-          nvm install 10
-          yarn --frozen-lockfile --ignore-engines
+          node-version: '10'
```

Comment:
Why go back to 10?

Reply:
I saw two conflicting versions specified: `11.0.0` for the github action, and `10` for `nvm`.

I figured the one that was effective was the one installed via nvm, so I kept version 10 not to change it in the context of this PR.